packages/frontend/src/components/ProjectConnection/WarehouseForms/SnowflakeForm.tsx

```diff
@@ -95,6 +97,16 @@ const SnowflakeForm: FC<{
         },
     });
 
+    const { data: orgCredentials } = useOrganizationWarehouseCredentials();
+    const snowflakeOrgCredentials =
```

Comment:
I thought about reusing in this form the new SnowflakeOauthInput, but this logic in this form is ab it more complex, so it was not easy to replace, perhaps in a separate PR